specs/language/introduction.tex

```diff
@@ -273,21 +273,52 @@
 
 \Sec{\acrshort{hlsl} Memory Models}{Intro.Memory}
 
-\p Memory accesses for \gls{sm} 5.0 and earlier operate on 128-bit slots aligned
-on 128-bit boundaries. This optimized for the common case in early shaders where
-data being processed on the GPU was usually 4-element vectors of 32-bit data
-types.
-
-\p On modern hardware memory access restrictions are loosened, and reads of
-32-bit multiples are supported starting with \gls{sm} 5.1 and reads of 16-bit
-multiples are supported with \gls{sm} 6.0. \gls{sm} features are fully
-documented in the \gls{dx} Specifications, and this document will not attempt to
-elaborate further.
+\p The fundamental storage unit in HLSL is a \textit{byte}, which is comprised
+of 8 \textit{bits}. Each \textit{bit} stores a single value 0 or 1. Each byte
+has a unique \textit{memory location}, alternatively called an \textit{address}.
+
+\p An object that is not of an intangible type will have an associated set of
```

Comment:
Can we avoid the double negative somehow?

```suggestion
\p An object that is of a tangible type will have an associated set of
```

Reply:
Can this ref the section on intangible types (if there is one)?

Reply:
I ended up removing this paragraph from this PR. I didn't actually define _object_ yet, so I think this should go in a section about the object model, not the memory model.